FirebaseMessaging.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'FirebaseMessaging'
-  s.version          = '4.6.2'
+  s.version          = '4.8.0'
```

Comment:
Should this be a major version bump, since it's removing an API?

Reply:
It will be 7.0.0, but maybe make it something like 4.99.0 for now to keep tests running.

Reply:
Done.